packages/core/stories/button.doc.stories.mdx

```diff
@@ -24,9 +24,12 @@ import {
 
 # Button
 
-A Button executes an action when clicked. Button labels should use only one or two words, and begin with an active verb. There are three Button variants: Primary is the most commonly-used button; Secondary performs an action less likely to be triggered; Call-To-Action (a.k.a. CTA) performs a critical action on the screen.
+A Button executes an action when clicked. Button labels should use only one or two words and begin with an active verb. There are three Button variants:  
+- **Primary** is the most commonly-used button.
```

Comment:
@sion-docs Do you think we should be consistent in the order of the variants when we refer to them?

Reply:
Yes, agreed.